packs/servicenow/actions/get.py

```diff
@@ -0,0 +1,11 @@
+from lib.actions import BaseAction
+
+
+class GetAction(BaseAction):
+    def run(self, table, query):
+        try:
+            self.client.table = table
+            response = self.client.get(query)
+            return response
+        except e:
```

Comment:
Minor style thing - this `try / except` is not needed if we don't do anything special with the exception (e.g. log it or throw a more granular exception) since the default behavior without try / except block is the same.